assets/lib/webinar-join.js

```diff
@@ -0,0 +1,75 @@
+window.onload = (event) => {
+  (function () {
+    document.querySelector(".d-header").style.display = "none";
+
+    ZoomMtg.preLoadWasm();
+    ZoomMtg.prepareJssdk();
+
+    const path = window.location.pathname;
+    const meetingId = path.split("/zoom/webinars/")[1].split("/sdk")[0];
+    let getParams = function (url) {
+      let params = {};
+      let parser = document.createElement("a");
+      parser.href = url;
+      let query = parser.search.substring(1);
+      let vars = query.split("&");
+      for (let i = 0; i < vars.length; i++) {
+        let pair = vars[i].split("=");
+        params[pair[0]] = decodeURIComponent(pair[1]);
+      }
+      return params;
+    };
+
+    let request = new XMLHttpRequest();
+    request.open("GET", `/zoom/webinars/${meetingId}/signature.json`, true);
+
+    request.onload = function () {
+      if (this.status >= 200 && this.status < 400) {
+        let res = JSON.parse(this.response);
+        console.log(res);
+        ZoomMtg.init({
+          leaveUrl: res.topic_url,
+          isSupportAV: true,
+          // audioPanelAlwaysOpen: false,
```

Comment:
Honestly I moved the code and wasn't sure about removing the commented code since it was there, it does nothing but thought maybe it was left for some reason, if you are okay with it I can remove it